AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -243,4 +249,16 @@ - (void)checkIfTokenNeedsToBeRefreshed:(MSAuthTokenValidityInfo *)tokenValidityI
   }
 }
 
+- (void)finishInitialize {
+  if (!self.resetAuthTokenRequired) {
```

Comment:
nit: can be simplified to
```
- (void)finishInitialize {
  if (self.resetAuthTokenRequired) {
      [self setAuthToken:nil withAccountId:nil expiresOn:nil];
  }
}
```

Reply:
@clpolet no. you missed `self.resetAuthTokenRequired = YES`

Reply:
That wasn't in the previous code due to the `return` statement. But I can still add it.